packages/block-library/src/gallery/use-mobile-warning.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { store as interfaceStore } from '@wordpress/interface';
+import { store as noticesStore } from '@wordpress/notices';
+
+export default function useMobileWarning() {
+	const { createWarningNotice } = useDispatch( noticesStore );
+	const { toggleFeature } = useDispatch( interfaceStore );
+	const isMobileWarningActive = useSelect( ( select ) => {
+		const { isFeatureActive } = select( interfaceStore );
+		return isFeatureActive( 'core/edit-post', 'mobileGalleryWarning' );
+	}, [] );
+
+	if ( ! isMobileWarningActive ) {
+		return;
+	}
+
+	createWarningNotice(
+		__(
+			'To edit this Gallery block in the WP mobile app you need to be using version 18.2 or higher.'
```

Comment:
```suggestion
			'Editing this Gallery Block in the WordPress mobile app requires version 18.2 or higher.'
```

Very subjective. Feel free to ignore. 

I think, at least, style required caps for blocks and WordPress for the product.

Also, would you think it's overboard to add a link to https://wp.com/app to upgrade? Might be too much.

Reply:
I like this updated wording (and using WordPress instead of WP). In both cases, there's an additional word hanging over onto another line. If we remove the word "block", it looks like we can get the notice onto a single line (in English at least):

| Before | After |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/14988353/140437647-191fd04e-7c88-4249-9b6b-83c75b2fd685.png) | ![image](https://user-images.githubusercontent.com/14988353/140437658-497f9318-b807-44f6-aa1d-3dd29097593e.png) |

But does that make it _too_ brief if we remove the additional word?